demo/index.html

```diff
@@ -9,161 +9,45 @@
-      padding: 20px;
-      border-radius: 8px;
-      background-color: white;
-      border: 1px solid #ddd;
-      max-width: 800px;
-      margin: 16px auto 0 auto;
-    }
-
-    .item:focus {
-      outline: 0;
-      border-color: #333;
-    }
-
-    .avatar {
-      height: 40px;
-      width: 40px;
-      border-radius: 20px;
-      box-sizing: border-box;
-      background-color: #DDD;
-    }
-
-    .pad {
-      padding: 0 16px;
-      @apply(--layout-flex);
-      @apply(--layout-vertical);
-    }
-
-    .primary {
-      font-size: 16px;
-      font-weight: bold;
-    }
-
-    .secondary {
-      font-size: 14px;
-    }
-
-    .dim {
-      color: gray;
-    }
-
-    .spacer {
-      @apply(--layout-flex);
-    }
-
-  </style>
-
+  <link rel="import" href="../../paper-styles/demo-pages.html">
 </head>
+<style>
+  a {
+    display: block;
+    margin-bottom: 10px;
+  }
+
+  li {
+    list-style: none;
+  }
+</style>
 <body unresolved>
-
-  <template is="dom-bind">
-    <iron-ajax url="data/contacts.json" last-response="{{data}}" auto></iron-ajax>
-
-      <app-header condenses fixed effects="resize-title blend-background waterfall">
-        <app-toolbar>
-          <paper-icon-button icon="menu" drawer-toggle></paper-icon-button>
-          <h4 condensed-title>iron-list <i>&mdash; Demo</i></h4>
-          <paper-icon-button icon="search"></paper-icon-button>
-          <paper-icon-button icon="more-vert"></paper-icon-button>
-        </app-toolbar>
-        <app-toolbar class="tall">
-          <h1 title>iron-list</h1>
-        </app-toolbar>
-      </app-header>
-
-      <!-- iron-list using the document scroll -->
-      <iron-list items="[[data]]" as="item" scroll-target="html">
-        <template>
-          <div>
-            <div class="item" tabindex$="[[tabIndex]]">
-              <iron-image class="avatar" sizing="contain" src="[[item.image]]"></iron-image>
-              <div class="pad">
-                <div class="primary">[[item.name]] [[index]]</div>
-                <div class="secondary">[[item.shortText]]</div>
-                <div class="secondary dim">[[item.longText]]</div>
-              </div>
-            </div>
-          </div>
-        </template>
-      </iron-list>
-
-  </template>
-
+  <div class="horizontal-section-container">
+    <div>
+      <h4>iron-list demos</h4>
+      <ul class="horizontal-section">
+        <li><a href="cards.html">List of cards</a></li>
+        <li><a href="collapse.html">Collapsible items</a></li>
+        <li><a href="scroll-threshold.html">Scroll thresholds</a></li>
+        <li><a href="grid.html">Grid layout</a></li>
+        <li><a href="selection.html">Contact selection</a></li>
+        <li><a href="basic.html">Basic list</a></li>
+      </ul>
+    </div>
+  </div>
+
+  <script>
+    if (window.innerWidth < 600) {
+      var base = document.createElement('base');
+      base.target = '_blank';
+      document.head.appendChild(base);
+    }
+  </script>

```

Comment:
it's a way to fix #258 without doing UA detection. Particularly, the demos look better without iframe on those devices. For example, selection.html has a drawer that would otherwise interact poorly with the drawer in the catalog.